install_config/topics/ocp_osp_prerequisites.adoc

```diff
@@ -7,22 +7,21 @@ regarding the prerequisites and configuration changes required for an {product-t
 OpenStack environment are discussed in detail.
 
 NOTE: All the OpenStack CLI commands in this reference environment are executed
-using the CLI `openstack` commands within the OpenStack director node. If using a
-workstation or laptop to execute these commands instead of the OpenStack
-director node, please ensure to install the following packages found
-within the specified repositories.
+using the CLI `openstack` commands from a different node than the OpenStack
```

Comment:
[NOTE]
All of the OpenStack CLI commands in this reference environment are executed using the CLI `openstack` commands within a different node from the director node. The commands are executed in the other node to avoid package conflicts with new versions of Ansible. Be sure to install the following packages in the specified repositories.